src/app/common/utils/system-notifications.service.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { TestBed } from '@angular/core/testing';
+import { SystemNotificationsService } from './system-notifications.service';
+
+describe('SystemNotificationsService', () => {
+  let service: SystemNotificationsService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(SystemNotificationsService);
+  });
+
+  it('should be created', () => {
```

Comment:
This one can be removed, it is not doing anything special. Instead you should add test for `showNotification` method :)